src/backend/access/appendonly/appendonly_blkdir_udf.c

```diff
@@ -0,0 +1,210 @@
+/*------------------------------------------------------------------------------
+ *
+ * AppendOnly_Blkdir UDFs
+ *   User-defined functions (UDF) for support of append-only block directory
+ *
+ * Copyright (c) 2013-Present VMware, Inc. or its affiliates.
```

Comment:
use cbdb's Copyright

Reply:
I found the Cloudberry Database team does not initially create or change this file, so we should keep the original copyright information unchanged.